setup.py

```diff
@@ -12,12 +12,13 @@
 
 install_requirements = [
     "boto3>=1.3,<2.0",
-    "click==6.7",
+    "click>=7.0<8.0",
     "PyYaml==3.13",
     "Jinja2>=2.8,<3",
-    "packaging==16.8",
-    "colorama==0.3.7",
-    "six==1.11.0"
+    "packaging>=18.0<19.0",
```

Comment:
I am not sure what the consequences of upgrading this package by 2 major version releases will be. Prefer to keep `>=16.0,<17.0` for now.

Reply:
`17.0` drops support for python 2.6 which we still support (according to our classifiers)